vuepress/docs/next/tutorials/ecr/deploy-components-without-bundle.md

```diff
@@ -1,46 +1,54 @@
 ---
 redirectFrom: /next/tutorials/ecr/tutorials/ecr-deploy-use-plugin-and-mfe-without-bundle.html
 ---
-# Use Blueprint Generated Plugin and Micro Frontends Without a Bundle
+# Apply Entando Blueprint Without a Bundle
 
 ## Overview
 
-In this tutorial you will learn how to use the plugin and microfrontend generated using the Entando Blueprint in a running cluster without the need to include the components in a bundle.
-
-**If you haven't already generated a plugin and micro frontends with the Component Generator go here first [Generate a Plugin](../backend-developers/generate-microservices-and-micro-frontends.md)**
+In this tutorial you will learn to apply an Entando Blueprint to a plugin and Micro Frontend generated with the Entando Component Generator in a running cluster, without the need for a bundle.
 
 ## Prerequisites
 
--   An Entando Plugin built with the Entando Component Generator and populated with micro frontends
+-   An Entando Plugin built with the Entando Component Generator and populated with Micro 
+Frontends. [Generate a plugin and Micro Frontend here](../backend-developers/generate-microservices-and-micro-frontends.md). 
+
+-   Node and NPM installed on your machine (use LTS version)
 
--   Node and NPM are installed on your machine (use LTS version)
+-   Docker installed on your machine and able to upload images to Docker Hub or an image repository of your choice
 
--   Docker is installed on your machine and you are able to upload images to docker-hub or an image repository of your choice
+-   An instance of the Entando platform running Kubernetes. See [Getting Started](../../docs/getting-started/README.md)
 
--   An installed instance of the Entando platform running Kubernetes. See [Getting Started](../../docs/getting-started/README.md)
+::: tip Note
+[Here is a definition of an Entando plugin,](../../docs/ecr/ecr-bundle-details.html#plugin) including information on the runtime contract required for using non-Java based plugins.
+:::
 
-## Steps
 
-## 1. Generate a docker image for your microservice
+## Generate a Docker image 
 
 JHipster uses the JIB Maven plugin to generate a docker image for your
 microservice.
 
-The name of the output image generated with JIB will be composed by:
-- The organization you chose during the setup wizard (by default that's set to `entando`)
+The name of the output image generated with JIB during the setup wizard is defined with:
+- The organization name you chose, default is set to `entando`
```

Comment:
should setup wizard be capitalized?
i think ; instead of , in "...you chose, default..."